lib/dox/formatters/plain.rb

```diff
@@ -2,7 +2,26 @@ module Dox
   module Formatters
     class Plain < Dox::Formatters::Base
       def format
-        body
+        normalize(body)
+      end
+
+      private
+
+      def normalize(data)
+        case data
+        when String
+          to_utf8(data)
+        else
```

Comment:
Can it be anything else but string?

Reply:
I don't think it can actually...will remove it 👍